.github/workflows/e2e-full.yml

```diff
@@ -55,17 +55,3 @@ jobs:
           name: playwright-report-uat
           path: packages/client/playwright-report/
           retention-days: 7
-
```

Comment:
start to wind back the relentless full E2E-ing we are doing atm (pushed directly to master), looks like the web credit tests are working consistently now (although I have done nothing to fix them .. anyway the UAT deployment is the same commit as Prod ..